variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "project_id" {
```

Comment:
The credentials file already contains project id.
Let's rely on that instead of having a variable.

Reply:
For `terraform-google-modules/network/google` module project_id is mandatory. 
Currently, it's taking from `data.google_compute_zones.available.project` (assuming **project_id** is set on provider block).

Reply:
That works!